powershell/plugins/plugin-create-inline-properties.ts

```diff
@@ -43,6 +44,16 @@ function getNameOptions(typeName: string, components: Array<string>) {
   return [...result.values()];
 }
 
+function getCombinedDescription(rawDescription: string, externalDocsUrl?: string, externalDocsDescription?: string): string {
+  let description = rawDescription ?? "";
+  if (undefined !== externalDocsUrl && "" !== externalDocsUrl) {
```

Comment:
May simplified to !externalDocsUrl

Reply:
I guess you mean !!externalDocsUrl

Reply:
![image](https://user-images.githubusercontent.com/19869716/197961384-d5a755dd-e528-4687-8f87-80c79ae2c891.png)